lib/themes-source/base.less

```diff
@@ -30,7 +30,7 @@
  * The picker input element.
  */
 .picker__input {
-    cursor: default;
```

Comment:
@amsul Is this a change you'd like to take in? Perhaps there's a reason why it was explicitly set as default before?